tests/spec_decode/test_multi_step_worker.py

```diff
@@ -418,3 +418,4 @@ def test_draft_proposals_mixed_k():
     assert proposals.proposal_lens.tolist() == [
         k for _ in range(expected_num_proposal_seqs - 1)
     ] + [0 for _ in range(expected_num_no_proposal_seqs)] + [k]
+>>>>>>> main:tests/spec_decode/test_multi_step_worker.py
```

Comment:
Is this intended? Otherwise, please fix this.

Reply:
I think it was accidently in when I resolved merge conflict